tools/authority_identity_key

```diff
@@ -0,0 +1,41 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Hum, do we want a private key in there?

Reply:
No, this one is used for development, and should not be used in practice. (Note that currently the signature validation for the consensus is disabled until I figure out why it does not work as intended.)

Reply:
So if it's just used for development, is it possible to create these keys locally on demand? So that they don't have to be included in the repo?

Reply:
Now that I am thinking about it more carefully, we absolutely need the certificate with an hardcoded v3ident for running the arti-rest test suite, but we only use the private key to generate the custom consensus with the python script.
So technically, we can generate a set of private key/certificate when testing the python script, but we can use a different private key for signing the certificates in arti-rest.